packages/animations/browser/src/render/transition_animation_engine.ts

```diff
@@ -292,10 +293,10 @@ export class AnimationTransitionNamespace {
   }
 
   deregister(name: string) {
-    delete this._triggers[name];
+    this._triggers.delete(name);
 
-    this._engine.statesByElement.forEach((stateMap, element) => {
-      delete stateMap[name];
+    this._engine.statesByElement.forEach((stateMap) => {
```

Comment:
nit: I think that in general here arrow functions with a single non-typed argument don't have brackets :slightly_smiling_face: 
```suggestion
    this._engine.statesByElement.forEach(stateMap => {
```

Reply:
Thanks!